packages/components/src/core/Button/index.tsx

```diff
@@ -23,6 +23,7 @@ type ButtonSize = "small" | "medium" | "large";
 interface BaseButtonProps extends RawButtonProps {
   children?: React.ReactNode;
   isAllCaps?: boolean;
+  target?: "_blank" | "_self" | "_parent" | "_top" | "framename";
```

Comment:
discussion: instead of hardcoding this, I wonder we should use an existing type? Like maybe `AnchorHTMLAttributes<HTMLAnchorElement>["target"];` 😄 ?

Reply:
Oh, sure! That makes perfect sense. 🥸